pkg/registry/rbac/clusterrole/storage/storage.go

```diff
@@ -30,6 +30,14 @@ type REST struct {
 	*genericregistry.Store
 }
 
+// Implement ShortNamesProvider
+var _ rest.ShortNamesProvider = &REST{}
+
+// ShortNames implements the ShortNamesProvider interface. Returns a list of short names for a resource.
+func (r *REST) ShortNames() []string {
+        return []string{"cr"}
```

Comment:
Thanks for the PR. But are these shortnames official ones?. Because otherwise, this PR breaks the commands relying on some CRD whose short name is `cr`, etc.